### PR TITLE
Added stop command for containers

### PR DESCRIPTION
**Pull request by cenouralm**
Closes https://github.com/inveniosoftware/invenio-cli/issues/109

### CODE REVIEW

File: tests/test_helpers/test_commands.py [hunk 1]
```diff
@@ -385,12 +385,18 @@ def test_containerizedcommands_containerize(
...
     commands.docker_helper.execute_cli_command.reset_mock()
 
-    commands.containerize(pre=True, force=True, install=True)
+    commands.containerize(pre=True, force=True, install=True, stop=False)
 
     assert commands.docker_helper.execute_cli_command.mock_calls == (
         expected_force_calls + expected_setup_calls
     )
 
+    # Case: pre=True, force=True, install=True, stop=True
```
Comment (fenekku) [inline tests/test_helpers/test_commands.py:394]:
```suggestion
    # Case: pre=False, force=False, install=False, stop=True
```

**Review by fenekku — APPROVED** [review]
Just an update to one of the comments. Otherwise good.

It might be worthwhile considering doing the same thing for the local commands eventually. Could you create a ticket for that?

**Comment by cenouralm** [comment]
> Just an update to one of the comments. Otherwise good.
> 
> Also, pro tip for your PR message "adresses issue number". Replace it with one of these: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword to automate the closing of the issue when the PR is merged.
> 
> It might be worthwhile considering doing the same thing for the local commands eventually. Could you create a ticket for that?

Will do! Thank you for the feedback!

**Comment by cenouralm** [comment]
Just created the issue #156. Is there a specific description I should add?

**Comment by fenekku** [comment]
that's fine by me :+1:

**Pull request merged**